### PR TITLE
ci: Add retry for fat container

### PR DESCRIPTION
**Pull request by yatinappsmith**
## Description

add rerun support for fat

## Type of change

- Bug fix (non-breaking change which fixes an issue)
- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

code review

## Checklist:

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: .github/workflows/integration-tests-command.yml [hunk 1]
```diff
@@ -106,8 +106,8 @@ jobs:
         uses: martijnhols/actions-cache/restore@v3
         with:
           path: |
-            ~/combined_failed_spec
-          key: ${{ github.run_id }}-"ui-test-result"-${{ steps.timestamp.outputs.timestamp }}
+            ~/combined_failed_spec_fat
+          key: ${{ github.run_id }}-"ui-test-result"
```
Comment (mohanarpit) [inline .github/workflows/integration-tests-command.yml:110]: Do we also want to change the key and make it specific for fat-container?

Comment (yatinappsmith) [reply .github/workflows/integration-tests-command.yml:110]: I am using the same key with different paths as "ui-test-result" will create combined failed spec for both the runs.

**Review by mohanarpit — APPROVED** [review]

**Pull request merged**